akka-persistence-typed/src/main/scala/akka/persistence/typed/AutoConfirmation.scala

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed
+
+import akka.Done
+import akka.actor.typed.ActorRef
+
+/**
+ * A `Done` response message is automatically sent when commands implement this trait.
+ * The reply is sent after events in the returned effects have been persisted.
+ * The reply is only sent if the effects contains events to be persisted. For example
+ * stashing a command and returning `Effect.none` will not send a reply until the
+ * command is unstashed and actually handled.
+ * The reply is sent to the given [[AutoConfirmation#replyTo]] `ActorRef`, which can be the `replyTo`
+ * for an `ask` request.
+ */
+trait AutoConfirmation {
```

Comment:
What's the discoverability of this feature?

Reply:
Close to zero.

An alternative I tried some time ago was to have a different `askWithConfirmation` method on the call site. https://github.com/akka/akka/pull/24964

I don't like that, either, since the responsibility of if the confirmation should be automatic or explicit shouldn't be dictated by the sender of the command. Imagine that the service implementation and the Entity are developed by different people.